octoprint_mrbeam/analytics/analytics_handler.py

```diff
@@ -147,13 +153,16 @@ def _days_passed(path_to_file):
 
 	def analytics_user_permission_change(self, analytics_enabled):
 		self._logger.info("analytics user permission change: analyticsEnabled=%s", analytics_enabled)
+		if self._analyticsOn is None:
```

Comment:
@Gallore Is this really the right thing here?
When the user agrees or disagrees for the first time, we start the uploader in any case?

Reply:
Changed to only activate upload when the analytics are accepted the first time